### ISSUE TITLE
"FileNotFoundException on Assembly WebEye.Stream.x64/32"/ .net core 3.1 using demo Stream.GetLocalStreams

### ISSUE THREAD

**Issue by professordave** [jacobbo/WebEye#73]
Using the nuget version 
WebEyeStreamControl.Wpf   1.0.3
WebEye.Stream      1.0.2

Hello,  Copied the demo code into a working .net core 3.1 project. Instead of my original real main window, place this code into another new Main windows that gets shown by a button. 

Click on button Add Local Stream
When the execution gets to the line from the demo:
 public ObservableCollection<Stream> LocalStreams { get; }
            = new ObservableCollection<Stream>(Stream.GetLocalStreams());  

It throws a System.IO.FileNotFOundException.

> System.IO.FileNotFoundException: 'Could not load file or assembly 'WebEye.Stream.x64, Version=0.0.0.0, Culture=neutral, PublicKeyToken=null'. The system cannot find the file specified.'

Tried forcing project to 32bit and 64bit and get the error for either the 64bit or the 32 bit assembly

I tried some code from the github repository
  var path = Path.Combine(AppDomain.CurrentDomain.BaseDirectory,
                Environment.Is64BitProcess ? "WebEye.Stream.x64.dll" : "WebEye.Stream.Win32.dll");
            var name = AssemblyName.GetAssemblyName(path);
            return Assembly.Load(name).GetType("WebEye.ManagedWrapper");     

And it seems to fail on the load, but the GetAssemblyName does succeed as the dll files are there

**Comment by jacobbo** [comment]
Hello, sorry, did not test it with .net core, try .net framework.

**Comment by professordave** [comment]
Yes the demo compiled and runs fine in .net 4.x

Problem is core 3.1

**Comment by professordave** [comment]
I have an existing .net core 3.1 wpf project that I would like to add some video capture from usb attached video camera

**Comment by professordave** [comment]
Is the NuGet package too old. I am trying to use the NuGet package.

**Comment by jacobbo** [comment]
1.0.3 is the latest version.

**Comment by arteny** [comment]
Same error for .NET Framework 4.5.2 (WebEye.Stream.x64) not found.
Where I can get WebEye.Stream.x64.dll?

**Comment by jacobbo** [comment]
the dll is part of the nuget package. do you see it there?

**Comment by arteny** [comment]
no, it is not present there for the mentioned framework version

Best regards,   Alexey    Barkalov
С уважением, Алексей Баркалов




________________________________
От: Alexander Iacobciuc <notifications@github.com>
Отправлено: 5 июля 2020 г. 2:09
Кому: jacobbo/WebEye <WebEye@noreply.github.com>
Копия: Alexey <root@arteny.ru>; Comment <comment@noreply.github.com>
Тема: Re: [jacobbo/WebEye] "FileNotFoundException on Assembly WebEye.Stream.x64/32"/ .net core 3.1 using demo Stream.GetLocalStreams (#73)


the dll is part of the nuget package. do you see it there?

**Comment by jacobbo** [comment]
It is there for net40, which is compatible with net 4.5.2. Can you go to your output directory and check whether  WebEye.Stream.x64.dll is in there?

**Comment by arteny** [comment]
no, it was absent in program output directory, now I switched app to .net 4.6.1 and it becomes working

Best regards,   Alexey    Barkalov
С уважением, Алексей Баркалов




________________________________
От: Alexander Iacobciuc <notifications@github.com>
Отправлено: 5 июля 2020 г. 14:26
Кому: jacobbo/WebEye <WebEye@noreply.github.com>
Копия: Alexey <root@arteny.ru>; Comment <comment@noreply.github.com>
Тема: Re: [jacobbo/WebEye] "FileNotFoundException on Assembly WebEye.Stream.x64/32"/ .net core 3.1 using demo Stream.GetLocalStreams (#73)


It is there for net40, which is compatible with net 4.5.2. Can you go to your output directory and check whether WebEye.Stream.x64.dll is in there?

**Comment by gigyjoseph** [comment]
> Same error for .NET Framework 4.5.2 (WebEye.Stream.x64 not found). Where I can get WebEye.Stream.x64.dll?

Need to build Native project in 64 bit mode. And mostly you need Boost C++ library also 64 bit build
